llama.cpp

```diff
@@ -686,31 +581,17 @@ struct llama_model_loader {
             use_mmap = false;
         }
         this->use_mmap = use_mmap;
-        for (llama_load_tensor & lt : tensors_map.tensors) {
-            lt.calc_all();
-        }
     }
 
     bool alignment_prevents_mmap() {
         for (const llama_load_tensor & lt : tensors_map.tensors) {
-            for (const llama_load_tensor_shard & shard : lt.shards) {
-                if (shard.file_off & 3) {
-                    return true;
-                }
+            if (lt.file_off & 3) {
+                return true;
```

Comment:
@ggerganov do we still have the problem that a tensor offset is not aligned? I believe it is not.

Reply:
It shouldn't happen with current models, only with pre-ggjt models.

Reply:
thank you. I will remove this check.